core/src/main/java/io/questdb/cairo/FullBwdDataFrameCursorFactory.java

```diff
@@ -47,4 +48,10 @@ public DataFrameCursor getCursor(SqlExecutionContext executionContext, int order
     public int getOrder() {
         return ORDER_DESC;
     }
-}
\ No newline at end of file
+
+    @Override
+    public void toPlan(PlanSink sink) {
+        sink.type("FullBwdDataFrame");
```

Comment:
Can we use reflection perhaps to output the class name except `CursorFactory` by default?